.github/workflows/shortlog.yml

```
@@ -0,0 +1,23 @@
+name: check contributor list
+
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    paths-ignore:
+      - '**'
+      - '!CONTRIBUTING.md'  # e.g. when new contributor adds themselves
+      - '!cylc/flow/__init__.py'  # e.g. when __version__ is changed for a new release
```

Following on from https://github.com/cylc/cylc-flow/pull/4205/files#r631898770, I think this could be simplified to
```suggestion
  workflow_dispatch:
  pull_request:
    paths:
      - 'CONTRIBUTING.md'
```